sdk/tables/Azure.Data.Tables/src/Generated/Operations/TableRestClient.cs

```diff
@@ -25,7 +25,7 @@ internal partial class TableRestClient
         private HttpPipeline _pipeline;
 
         /// <summary> Initializes a new instance of TableRestClient. </summary>
-        public TableRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string url, string version = "2018-10-10")
+        public TableRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string url, string version = "2019-02-02")
```

Comment:
What version do we have in out ClientOptions?

Reply:
good catch!